music.js

```diff
@@ -1,17 +1,39 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
 
-  // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.playcount = 0;
+
+
+// When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
+
+  Artist.prototype.newSong = function (title) {
```

Comment:
No need for a `prototype` method here. Inside the `Song` function constructor, you can simply do:

`this.artist.songs.push(this)`